data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+coala 2017 Application Template
+===============================
+
+```
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Which country will you reside in during the summer?
+- Which city (or other geographical boundary you feel comfortable sharing)?
+- Time Zone:
+- GSoC blog RSS feed URL: (You will have to write a blog post every
+  week during GSoC.)
+
+
+Code Sample
```

Comment:
https://resume.github.com is another service that could be used to provide the broad 'other' information, so that the applicant can focus on their five best.

https://github.com/resume/resume.github.com